nubison_model/Model.py

```diff
@@ -18,20 +18,52 @@ def load_model(self) -> None: ...
     def infer(self, input: any) -> any: ...
 
 
+def _is_shareable(package: str) -> bool:
+    # Nested requirements, constraints files, local packages, and comments are not supported
+    if package.startswith(("-r", "-c", "-e .", "-e /", "/", ".", "#")):
```

Comment:
Isn't it possible to have cases that meet the false criteria outside of the if statement as well?

Reply:
Good question! I reviewed [PEP 508](https://peps.python.org/pep-0508/) and considered potential cases to cover. It would be great if you could identify any additional cases or suggest a method to validate them.